next-tavla/src/Admin/scenarios/Boards/index.tsx

```diff
@@ -36,13 +36,15 @@ import {
     restrictToHorizontalAxis,
     restrictToWindowEdges,
 } from '@dnd-kit/modifiers'
+import { FilterButton } from './components/FilterButton'
 
 function Boards({ boards }: { boards: TBoard[] }) {
     const [settings, dispatch] = useReducer(settingsReducer, {
         search: '',
         sort: { type: 'descending', column: 'lastModified' },
-        columns: ['name', 'url', 'actions', 'lastModified'],
+        columns: ['name', 'url', 'tags', 'actions', 'lastModified'],
         boards: boards,
+        filterTags: [],
```

Comment:
Synes denne er grei for nå, men om det blir flere filtere bør vi kanskje vurdere å lage et eget filter-objekt, typ
```
filters: {
    search: '',
    tags: [],
    transportModes: [],
}
```